src/scss/custom/components/_footer.scss

```diff
@@ -0,0 +1,164 @@
+footer {
```

Comment:
I'm not super sure about this one. I guess it depends on if we are planning to force all our MoFo sites use the same footer (styling-wise) or not. I ***think*** we should make this an unified standard and let exceptions override the rules if necessary. 

Thoughts? /cc @kristinashu @taisdesouzalessa @alanmoo @gvn